applications/launchpad/gui-react/src/components/HelpTip/index.tsx

```diff
@@ -16,27 +16,29 @@ import { HelpTipProps } from './types'
  * @prop {CSSProperties} [style] - styles to apply to main wrapper element
  * @prop {boolean} [header] - whether the help tip should be rendered with additional top/bottom margin suitable for headers
  */
-const HelpTip = ({ text, cta, onHelp, style, header }: HelpTipProps) => (
```

Comment:
I think you can revert back to `() => ()` from `() => { return () }`

Reply:
Fixed.